.gitignore

```diff
@@ -6,3 +6,4 @@ VERSION
 *.rbc
 *.swp
 /.bundle
+.idea/
```

Comment:
`.idea`? A directory for "Intellij IDE"?
I am not know detail about it.
Kind of this case `**/.idea` is possible?
`/.idea` looks better, isn't it referring `/.bundle`?